README.md

```diff
@@ -74,6 +79,48 @@ environment before installing from the environment.yml.
 1. Run the tests.
 
 
+## Adding a new test suite:
+
+1. Create a subdirectory within `test` that contains a notebook, environment,
+   version and supporting files, as described in the next section. For ease, it
+   is simplest to use the same string for the subdirectory name and the suite
+   name.
+1. Update the `test/Makefile` to be able to build a Docker image for the new
+   test suite:
+
+   ```
+   <new-suite-name>-image
+       docker build -t ghcr.io/nasa/regression-tests-<new-suite-name>:latest -f ./Dockerfile --build-arg notebook=<new-test-notebook-name> --build-arg sub_dir=<new-suite-subdirectory> .
+   ```
+1. Update the `make images` rule to include building the new image.
+
+   ```
+   images: <pre existing rules already listed> <new-suite-name>-image
+   ```
+1. Update `test/run_notebooks.sh` to include the new test image in `all_images`:
+   ```
+   all_images=(<pre existing test suites> <new-suite-name>)
+   ```
+
+With this in place, the new test suite should be able to be built and run:
+
+```bash
```

Comment:
You have to set the env and this won't let you test against local.  That should be spelled out probably.

Maybe describe creating the test suite?  with the environment and running the notebook against the local HIAB.

Reply:
> You have to set the env and this won't let you test against local. That should be spelled out probably.

Good call, will do.

> Maybe describe creating the test suite? with the environment and running the notebook against the local HIAB.

Yeah - maybe I was a bit lazy here as some of this is covered in the section below. I'll improve this now.

Reply:
You can always just point them to the correct location.

Reply:
Okay - I think that's updated now. I added a reference to HIAB, and also went into more detail on the contents of the subdirectory for each test suite.